src/main/kotlin/com/openlattice/chronicle/services/studies/StudyService.kt

```diff
@@ -259,13 +259,17 @@ class StudyService(
             SELECT * FROM ${STUDY_PARTICIPANTS.name} WHERE ${STUDY_ID.name} = ?
         """.trimIndent()
 
-        private val COUNT_LEGACY_STUDY_ID_SQL = """
-            SELECT count(*) FROM ${LEGACY_STUDY_IDS.name} WHERE ${LEGACY_STUDY_ID.name} = ?
-        """.trimIndent()
-
-        private val SELECT_REAL_STUDY_ID_SQL = """
-            SELECT ${STUDY_ID.name} FROM ${LEGACY_STUDY_IDS.name} WHERE ${LEGACY_STUDY_ID.name} = ?
-        """.trimIndent()
+        private fun selectStudyIdSql(table: String) :String {
+            return when (table) {
+                LEGACY_STUDY_IDS.name -> """
+                    SELECT ${STUDY_ID.name} FROM ${LEGACY_STUDY_IDS.name} WHERE ${LEGACY_STUDY_ID.name} = ?                    
+                """.trimIndent()
+                STUDY_ID.name -> """
+                    SELECT ${STUDY_ID.name} FROM ${STUDIES.name} WHERE ${STUDY_ID.name} = ?
+                """.trimIndent()
```

Comment:
there is no table named `STUDY_ID`

Reply:
```
private fun selectStudyIdsSql(table: String, whereClauseColumnName: String) {
   return SELECT ${STUDY_ID.name} from table where whereClauseColumnName = ?
}
```

Reply:
I'm selecting from `STUDIES.name` not `STUDY_ID.name`.